<!--REQUEST_ID=4c0990ef-b4e2-49c9-bb01-eef84c896e62-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48362&attempt_id=8f3b2645-e418-45af-8f3b-6ef838727ef4)

Attempted: spectmorph

<details><summary>Partial log (click to expand)</summary><p>

```
       float *spectrum = reinterpret_cast <float *> (&out[0]);
                                                      ^~~
                                                      putw
make[2]: *** [Makefile:653: smnoisedecoder.lo] Error 1
make[2]: Leaving directory '/build/spectmorph-0.4.1/lib'
make[1]: *** [Makefile:537: all-recursive] Error 1
make[1]: Leaving directory '/build/spectmorph-0.4.1'
make: *** [Makefile:435: all] Error 2
builder for '/nix/store/kzyg442jj3icalpp2r6yd89rbhyxlf6a-spectmorph-0.4.1.drv' failed with exit code 2
error: build of '/nix/store/kzyg442jj3icalpp2r6yd89rbhyxlf6a-spectmorph-0.4.1.drv' failed
```
</p></details>

--- a/pkgs/applications/audio/spectmorph/default.nix
+++ b/pkgs/applications/audio/spectmorph/default.nix
@@ -1,0 +1,22 @@
+{ stdenv, fetchurl, pkgconfig, libjack2, lv2, glib, qt5, libao, cairo, libsndfile, fftwFloat }:
+
+stdenv.mkDerivation rec {
+  name = "spectmorph-${version}";
+  version = "0.4.1";
+  src = fetchurl {
+    url = "http://spectmorph.org/files/releases/${name}.tar.bz2";
+    sha256 = "0z00yvv3jl8qsx6bz9msmg09mdnj5r5d4ws5bmnylwxk182whbrv";
+  };
+
+  buildInputs = [  libjack2 lv2 glib qt5.qtbase libao cairo libsndfile fftwFloat ];
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  meta = with stdenv.lib; {
+    description = "Allows to analyze samples of musical instruments, and to combine them (morphing) to construct hybrid sounds";
+    homepage = http://spectmorph.org;
+    license = licenses.gpl3;
+    platforms = [ "x86_64-linux" "i686-linux" ];
+    maintainers = [ maintainers.magnetophon ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17101,6 +17101,8 @@ with pkgs;
 
   singularity = callPackage ../applications/virtualization/singularity { };
 
+  spectmorph = callPackage ../applications/audio/spectmorph { };
+
   spectrwm = callPackage ../applications/window-managers/spectrwm { };
 
   spectral = qt5.callPackage ../applications/networking/instant-messengers/spectral { };

